xscen/extract.py

```diff
@@ -1223,6 +1240,36 @@ def _restrict_multimembers(catalogs: dict, id_columns: list, restrictions: dict)
     return catalogs
 
 
+def _restrict_wl(catalogs: dict, restrictions: dict):
+    """Update the results from search_data_catalogs by removing simulations that are not available in the warming level csv."""
+    df = pd.concat([catalogs[s].df for s in catalogs.keys()])
```

Comment:
Following my other comment on `catalog` instead of `catalogs`, this line won't be needed.